### PR TITLE
Adding GitHub PR template to the repo

### PR DESCRIPTION
**Pull request by davidffrench**
## Motivation
No ticket - It can be very useful to have a PR template in GitHub to ensure all the details of the change are captured. This can be very useful when looking back at older issues and having all the context in the PR.

## What
Add GitHub PR template

## Why
Provide additional context on PRs

## How
Adding a PULL_REQUEST_TEMPLATE.md to a .github folder at the root of the project

## Verification Steps
None

## Checklist:
- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task
- [ ] TODO

### CODE REVIEW

**Comment by mindreeper2420** [comment]
@davidffrench there is an issue for this, with a discussion on requirements for a PR Template: https://github.com/integr8ly/tutorial-web-app/issues/164

**Comment by davidffrench** [comment]
Thanks @mindreeper2420 . I'll put a comment on that issue with this template, including some of the details from your suggestion also.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ 
+1. Go to `XX >> YY >> SS`
+2. Create a new item `N` with the info `X`
+3. Try to edit this item 
+4. Check if in the left menu the feature X is not so long present.
+
+## Checklist:
+
+- [ ] Code has been tested locally by PR requester
+- [ ] Changes have been successfully verified by another team member
```
Comment (dlabaj) [inline .github/PULL_REQUEST_TEMPLATE.md]: Do we need this?  PR has to be approved by 1 person to have it committed.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+## Verification Steps
+Add detailed steps required to check this change. Following an example.
+ 
+1. Go to `XX >> YY >> SS`
+2. Create a new item `N` with the info `X`
+3. Try to edit this item 
+4. Check if in the left menu the feature X is not so long present.
+
+## Checklist:
+
```
Comment (dlabaj) [inline .github/PULL_REQUEST_TEMPLATE.md]: Wondering if we should have [ ] Screen shots included (if applicable)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (davidffrench) [reply .github/PULL_REQUEST_TEMPLATE.md]: @dlabaj This should include what steps make sense for the process on PRs on your team and this repo. These are just what we use on my team, it is required that another team member has not only reviewed the code but verified the fix through verification steps either locally or more ideally on a cluster. 

Do ye have a process like this for this web app?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (davidffrench) [reply .github/PULL_REQUEST_TEMPLATE.md]: I really like the idea of reminding the individual creating the PR to include screenshots. I will add this now.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (dlabaj) [reply .github/PULL_REQUEST_TEMPLATE.md]: okay we can leave this then and see how it goes.

**Pull request merged**